docs/main/user-guides/replicated-storage-user-guide/replicated-pv-mayastor/openebs-on-kubernetes-platforms/gke.md

```diff
@@ -0,0 +1,259 @@
+mayastor-io-engine-6hv77                        2/2     Running   0          34s
+mayastor-io-engine-h96xf                        2/2     Running   0          14s
+mayastor-io-engine-ndstm                        2/2     Running   0          24s
+mayastor-localpv-provisioner-54c8bc9f94-khxgp   1/1     Running   0          3m56s
+mayastor-loki-0                                 1/1     Running   0          3m55s
+mayastor-nats-0                                 3/3     Running   0          3m56s
+mayastor-nats-1                                 3/3     Running   0          3m55s
+mayastor-nats-2                                 3/3     Running   0          3m55s
+mayastor-obs-callhome-57d977df6f-hnzn9          2/2     Running   0          3m56s
+mayastor-operator-diskpool-5b7df8f8cb-c559h     1/1     Running   0          3m56s
+mayastor-promtail-2srpn                         1/1     Running   0          3m57s
+mayastor-promtail-5qv27                         1/1     Running   0          3m57s
+mayastor-promtail-vvdpl                         1/1     Running   0          3m57s
+```
+
+## Pools
+
+The available GKE local SSD disks on worker nodes can be viewed by using the `kubectl-mayastor` plugin.
+
+```
+$ kubectl mayastor get block-devices gke-gke-ssd-default-pool-2a0f964a-hv99
+ DEVNAME       DEVTYPE  SIZE    AVAILABLE  MODEL       DEVPATH                                              MAJOR  MINOR  DEVLINKS 
+ /dev/nvme1n1  disk     375GiB  yes        nvme_card0  /devices/pci0000:00/0000:00:04.0/nvme/nvme1/nvme1n1  259    0      "/dev/disk/by-id/google-local-nvme-ssd-0", "/dev/disk/by-id/nvme-nvme.1ae0-6e766d655f6361726430-6e766d655f6361726430-00000001", "/dev/disk/by-id/nvme-nvme_card0_nvme_card0", "/dev/disk/by-id/nvme-nvme_card0_nvme_card0_1", "/dev/disk/by-path/pci-0000:00:04.0-nvme-1" 
+```
+
+:::important
+It is highly recommended to specify the disk using a unique device link that remains unaltered across node reboots. Examples of such device links are: by-path or by-id.
+:::
+
+The block size of the disks should be specified in accordance with the block size of the local SSD in your GKE. Run the following commands from the worker node to find the SSD block size:
+
+**Command**
+
+```
+fdisk -l /dev/nvme1n1
+```
+
+**Output**
+
+```
+root@gke-gke-ssd-default-pool-2a0f964a-980h:~# fdisk -l /dev/nvme1n1
+Disk /dev/nvme1n1: 375 GiB, 402653184000 bytes, 98304000 sectors
+Disk model: nvme_card0
+Units: sectors of 1 * 4096 = 4096 bytes
+Sector size (logical/physical): 4096 bytes / 4096 bytes
+I/O size (minimum/optimal): 4096 bytes / 4096 bytes
+```
+
+**Command**
+
+```
+lsblk -o NAME,PHY-SeC
+```
+
+**Output**
+
+```
+root@gke-gke-ssd-default-pool-2a0f964a-980h:~# lsblk -o NAME,PHY-SeC
+NAME         PHY-SEC
+nvme0n1          512
+├─nvme0n1p1      512
+├─nvme0n1p14     512
+└─nvme0n1p15     512
+nvme1n1         4096
+root@gke-gke-ssd-default-pool-2a0f964a-980h:~# 
+```
+
+### Pool.yaml
```

Comment:
Don't see any content for pool creation, just seeing the yaml

Reply:
Will update the section